src/solvers/refinement/string_refinement.cpp

```diff
@@ -1874,15 +1973,22 @@ static exprt instantiate(
 
 /// Instantiates a quantified formula representing `not_contains` by
 /// substituting the quantifiers and generating axioms.
+/// \param stream: a message stream
```

Comment:
Update this by removing unused arguments and updating `index_set` (and removing `current_index` set).